core/src/main/java/org/camunda/bpm/engine/test/assertions/AssertionsLogger.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.test.assertions;
+
+import org.camunda.bpm.engine.runtime.ActivityInstance;
+import org.camunda.bpm.engine.runtime.TransitionInstance;
+import org.camunda.commons.logging.BaseLogger;
+
+/**
+ * @author Ingo Richtsmeier
+ *
+ */
+public class AssertionsLogger extends BaseLogger {
+  
+  public static final AssertionsLogger INSTANCE = BaseLogger.createLogger(
+      AssertionsLogger.class, "BPM_ASSERT", "org.camunda.bpm.engine.test.assertions", "00");
+  
+  public void collectTransitionInstances(ActivityInstance activityInstance) {
+    logDebug("001", "collecting transition instances for activity instance '{}' with ID '{}'", 
+        activityInstance.getActivityName(),
+        activityInstance.getActivityId());
+  }
```

Comment:
The log message is a bit misleading. The activity instance is uniquely identified by its ID. I would log that instead of the activity name or change the content of the log message such that it is clear that the name is logged. The same applies to the transition instance logging.